templates/base.html

```diff
@@ -51,13 +51,12 @@
 		</div><!-- /.container for main content -->
 
 		<footer class="footer">
-				<div class="container">
+				<div class="container">	
 					<a href="https://github.com/CodeCatz">
-					<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<p>GITHUB CodeCatz: Kittens learning to code like Catz</p>
 					</a>
-				</div>
+				</div>							
```

Comment:
no, it's not done, at least not in this pull request :wink: 
any additional commits you push into your repository, will be added to this pull request

so you might not want to work on other stuff before this get's merged :smile: 
we'll have to show you how to create feature branches, I guess... :thought_balloon: